### PR TITLE
Removing _testing mock modules.

### PR DESCRIPTION
**Pull request by dhermes**
No longer needed since gRPC works (mostly?) everywhere.

---

@tseaver I stopped short of combining `REQUIREMENTS` and `GRPC_EXTRAS` in `setup.py` but it seems like that is do-able now. WDYT?

### CODE REVIEW

**Comment by dhermes** [comment]
Also think we can remove the [`bigtable` warning][1] and the scary warnings in the docs

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/6896144def9d0c8acd532451766ee293d5fb627a/gcloud/bigtable/__init__.py#L35-L38

File: scripts/update_json_docs.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -e
 
 function buildDocs () {
   echo "Building JSON documentation..."
-  PYTHONPATH=_testing python scripts/generate_json_docs.py --tag ${1}
+  python scripts/generate_json_docs.py --tag ${1}
```
Comment (daspecster) [inline scripts/update_json_docs.sh]: I tested this and it seemed to work.

Comment (dhermes) [inline scripts/update_json_docs.sh]: You mean without `_testing`?

**Comment by tseaver** [comment]
LGTM. [Push the button, Max!](https://www.youtube.com/watch?v=R3Igz5SfBCE).

**Comment by daspecster** [comment]
[2:09 is the result ;)](https://youtu.be/R3Igz5SfBCE?t=2m9s)

**Comment by dhermes** [comment]
Here's hoping it doesn't break AppVeyor: https://ci.appveyor.com/project/dhermes/gcloud-python/build/1.0.574.master

**Comment by tseaver** [comment]
W00t!  All green!

**Pull request merged**